packages/core/src/browser/preferences/preference-service.ts

```diff
@@ -70,7 +70,7 @@ export interface PreferenceService extends Disposable {
     readonly ready: Promise<void>;
     get<T>(preferenceName: string): T | undefined;
     get<T>(preferenceName: string, defaultValue: T): T;
-    get<T>(preferenceName: string, defaultValue: T, resourceUri: string): T;
+    get<T>(preferenceName: string, defaultValue: T, resourceUri?: string): T;
```

Comment:
I updated the typings as I believe they are incorrect. If a `defaultValue` is present, calling the method should not return `undefined` but rather the `defaultValue` if a preference cannot be found for a given `resourceUri`.